app/_kong_plugins/response-transformer-advanced/index.md

```diff
@@ -32,4 +32,40 @@ categories:
 
 search_aliases:
   - response-transformer-advanced
----
\ No newline at end of file
+
+related_resources:
+  - text: Response Transformer plugin
+    url: /plugins/response-transformer/
+---
+
+{% include plugins/request-response-transformer/response-transformer-description.md %}
+
+The Response Transformer Advanced plugin provides features not available in the [Response Transformer plugin](/plugins/response-transformer/), including:
+* When transforming a JSON payload, transformations are applied to nested JSON objects and
+  arrays. This can be turned off and on using the [`config.dots_in_keys`](./reference/#schema--config-dots-in-keys) configuration parameter.
```

Comment:
```suggestion
  arrays. This can be turned enabled and disabled using the [`config.dots_in_keys`](./reference/#schema--config-dots-in-keys) configuration parameter.
```